src/gpuarray_buffer_cuda.c

```diff
@@ -1305,6 +1305,10 @@ static int cuda_callkernel(gpukernel *k, unsigned int n,
     }
 
     switch (n) {
+    case 0:
+      ctx->err = cuLaunchKernel(k->k, 1, 1, 1, 1, 1, 1, shared,
+                                ctx->s, args, NULL);
+      break;
```

Comment:
The pattern is clear but running a 1, 1, 1 kernel is almost always a mistake and I don't want to encourage it.